runtime/parachains/src/inclusion_inherent.rs

```diff
@@ -106,6 +107,14 @@ decl_module! {
 
 			<scheduler::Module<T>>::schedule(freed);
 
+			// Temporarily mutably borrow this list so it can be truncated, then restore immutability.
+			// This only works because we own the vector anyway.
+			let backed_candidates = {
+				let mut backed_candidates = backed_candidates;
+				limit_backed_candidates::<T>(&mut backed_candidates);
+				backed_candidates
+			};
```

Comment:
```suggestion
			let backed_candidates = limit_backed_candidates::<T>(backed_candidates);
```

The comment doesn't bring any value. If you make the function take the value by value, it will be much cleaner code.

Reply:
174f9064